view/frontend/web/js/data/translations.json

```diff
@@ -0,0 +1,160 @@
+{
+  "en_US": {
+    "general": {
+      "nextStep": "Next Step",
+      "back": "Back",
+      "order": "Place order",
+      "cancel": "Cancel",
+      "saveAddress": "Save Address"
+    },
+    "steps": {
+      "shipping": "Shipping Step",
+      "payment": "Review & Payments Step"
+    },
+    "shipping": {
+      "title": "Shipping address",
+      "createAccount": "You can create an account after checkout",
+      "logIn": "You already have an account with us. Sign in <a href='%{link}'>here</a> or continue as guest."
+    },
+    "formElements": {
+      "firstName": "First name",
+      "lastName": "Last name",
+      "phoneNumber": "Phone Number",
+      "streetAddress": "Street Address",
+      "country": "Select Country",
+      "city": "City",
+      "zipCode": "Zip/Postal Code",
+      "state": "State/Province",
+      "selectState": "Select State/Province",
+      "company": "Company"
+    },
+    "payment": {
+      "title": "Billing Address",
+      "billingCheckbox": "My billing and shipping address are the same",
+      "editAddress": "Edit Address"
+    },
+    "success": {
+      "finished": "Thank you for your purchase!",
+      "orderNumber": "Your order number is: %{orderId}.",
+      "emailDetails": "We'll email you an order confirmation with details and tracking info.",
+      "continueShopping": "Continue shopping"
+    },
+    "discount": {
+      "title": "Discount",
+      "apply": "Apply Discount Code",
+      "addDiscount": "Apply Discount",
+      "removeDiscount": "Remove Discount"
+    },
+    "totals": {
+      "summary": "Order summary",
+      "subtotal": "Cart subtotal:",
+      "shipping": "Shipping:",
+      "tax": "Tax:",
+      "discount": "Discount:",
+      "orderTotals": "Order Total:"
+    },
+    "paymentMethods": {
+      "title": "Payment methods"
+    },
+    "shippingMethods": {
+      "title": "Shipping Methods"
+    },
+    "productList": {
+      "title": "Product List",
+      "name": "Product name:",
+      "price": "Price:",
+      "details": "Product details:",
+      "qty": "Qty:",
+      "discount": "Discount:"
+    },
+    "errorCode": {
+      "required": "This field is required!",
+      "generalError": "Looks like there was a problem:",
+      "unsupportedPaymentsForCountry": "In this country we don't handle any payment methods.",
+      "unsupportedShippingForCountry": "In this country we don't handle any shipping methods.",
+      "discountNotFound": "Coupon code not found! Failed when trying to activate: '%{code}' coupon code. Please use another one.",
+      "discountAddGeneralError": "Something goes wrong when trying to send coupon code. Please try again later.",
+      "discountDeleteError": "Failed when trying to delete: '%{code}' coupon code. Coupon code not found!",
+      "discountDeleteGeneralError": "Something goes wrong when trying to delete coupon code. Please try again later."
```

Comment:
Move errors related with discount to `discount` node. Please keep here just general errors required, min, max etc.

Reply:
Now this structure is flat and I'm using sentences in json obj